test/coverage.yaml

```diff
@@ -26,6 +26,7 @@ directories:
   source/common/tracing: 95.8
   source/common/watchdog: 60.0  # Death tests don't report LCOV
   source/exe: 94.4  # increased by #32346, need coverage for terminate_handler and hot restart failures
+  source/extensions/access_loggers/filters/cel: 95.7
```

Comment:
What can we do to increase test coverage back above the limit? Which lines are uncovered by current tests?